makepkg: refresh (#7499)
tldr-pages/tldr: pages/linux/makepkg.md

# makepkg

> Creates a package which can be used with `pacman`.

> > Uses the `PKGBUILD` file in the current working directory.

Reviewer: Should we mention `current working dir` here ?
It does take an option `-p` where we can supply any `PKGBUILD` file.
How about:
```suggestion
> Uses the `PKGBUILD` file in the current working directory by default.
```

> > Uses the `PKGBUILD` file in the current working directory by default.

Reviewer: I think this reads fine to me. Thanks @CleanMachine1

> > Creates a package which can be used with `pacman`.
> > Uses the `PKGBUILD` file in the current working directory by default.

Reviewer 2: I think this sounds better
```suggestion
> Create a package which can be used with `pacman`.
> Use the `PKGBUILD` file in the current working directory by default.
```

Author: I agree with the first line however the second line change seems out of place.

Reviewer: Yep, reading it on a new day, it does seem odd :(
We can maybe omit the `by default`, given we start with `Use`.

Reviewer 3: I also agree with @CleanMachine1.

Reviewer: Maybe this sounds better ? (thoughts)
```suggestion
> Creates a package which can be used with `pacman`.
> Uses the `PKGBUILD` file in the current working directory to build.
```

Author: I think axel's edited suggestion is sufficient
